config/development.json

```diff
@@ -153,6 +153,8 @@
 		"upgrades/precancellation-chat": true,
 		"upgrades/presale-chat": true,
 		"webpack/persistent-caching": false,
+		"woocommerce/extension": true,
+		"woocommerce/extension/stats": true,
```

Comment:
I'm not 100% sure how I feel about this. When I put in the `woocommerce/extension` part of this, it was in lieu of creating a feature flag for each extension, which would mean this line has a 1:1 relationship with the extension itself. If we include another line for a different part of the same extension, I'm not sure how that will work when we actually have feature flags for extensions.